internal/access/credential.go

```diff
@@ -225,3 +230,30 @@ func checkPasswordRequirements(db data.ReadTxn, password string) error {
 
 	return nil
 }
+
+// checkBadPasswords checks if the password is a known bad password, i.e. a widely reused password.
+func checkBadPasswords(password string) error {
+	badPasswordsFile := os.Getenv("INFRA_SERVER_BAD_PASSWORDS_FILE")
```

Comment:
This looks like it should be documented somewhere

Reply:
What do you think is the best way to document this? It's not a standard configuration value so it can't be configured through command line parameters or configuration file. This is mostly because I don't want to propagate the config all the way from `cmd` or store it in the database.

Reply:
A suggestion: move this out of the `access` layer and into the handler where it is used. There you'd have access to the server options which would provide an easy way to read this as a flag e.g. `--bad-passwords-file`

This new file based approach is elegant. Good stuff.

Reply:
Thing is, I don't really want to expose it to server options. Even if it is configurable from server options, the check should done here so it'll need to pass at least the value around

Reply:
Ah that's right, this is called from within in the `access` package. RE my comment: this is okay for now until we revisit the access layer separately